metadata-ingestion/docs/sources/azure-data-factory/azure-data-factory_pre.md

```diff
@@ -0,0 +1,209 @@
+## Overview
```

Comment:
@askumar27 take a look at the Vercel deployment of the docs site for this page [here](https://docs-website-a4y8880fz-acryldata.vercel.app/docs/generated/ingestion/sources/azure-data-factory) -  there's some unexpected behavior likely due to how we're auto-generating docs:

- `Config Details` shows up twice
- `Important Capabilities` and `Capabilities` sections are redundant
- `Quickstart Recipe` and `Starter Recipe` -- do we need both? 
- `Support Status: Incubating` this tag should be at the top of the page; currently showing up after the `Questions` section